app/containers/PrivacyPolicy/LeftMenu.jsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import * as routes from 'routes-config';
+
+import { getQuestionCode, getSectionCode } from 'utils/privacyPolicyManagement';
+
+import {
+  BORDER_PRIMARY_DARK,
+  BG_TRANSPARENT,
+  BORDER_TRANSPARENT,
+} from 'style-constants';
+
+import { ABase, FixedContentStyled } from 'containers/LeftMenu/FixedContent';
+import { LEFT_MENU_ID } from 'containers/LeftMenu/constants';
+
+import A from 'components/A';
+
+const A1 = ABase.extend`
+  ${({ route }) =>
+    (window.location.pathname + window.location.hash).match(route)
+      ? `
+    background-color: rgba(53,74,137,0.11);
```

Comment:
Should this color be in constants as well?

Reply:
sure